OpenTrons/opentrons-api
# fix(app): rename robot slideout character limit and error message

# Overview



Update strings and input validation on the Rename Robot slideout to match what's in Figma.

# Test Plan



- Go to Robot settings > Advanced > Rename robot
  - check that the input error appears on the 18th character of input
  - check for no regressions with invalid character recognition with ≤17 characters
  - compare text to Figma for "Slideouts/Modals"

# Changelog



- updated string for max characters
- new string for error (so it doesn't duplicate body text when triggered)
- updated tests

# Review requests



I think everything is completely contained to this one desktop slideout, but make sure nothing unintended has changed on ODD.

# Risk assessment



low, just strings and a tiny associated piece of validation logic.
